app/client/src/utils/Analytics/smartlook.ts

```
@@ -0,0 +1,27 @@
+      this.initialised = true;
+    }
+  }
+
+  public static identifyUser(userId: string, email: string) {
+    if (this.initialised) {
+      smartlookClient.identify(userId, {
+        email,
+      });
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)